src/app/common/modals/userinput.tsx

```diff
@@ -44,22 +43,6 @@ export const UserInputModal = (userInputRequest: UserInputRequest) => {
         [userInputRequest]
     );
 
-    function handleTextFocus() {
```

Comment:
For now, if the user is not focused on the user input modal, they can still remove it or confirm it with escape or enter

I think this is the correct behavior, because we wouldn't want the keybindings to unregister and then have the user be able to access the layer below

I briefly thought of disabling rather than unregistering the keybindings if they're not in focus, and I can do that in the future if we want, but I think for now it makes sense not to disable them, because the modal is the only thing on the screen anyway